torch/nn/functional.py

```diff
@@ -3835,8 +3835,8 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
             output_size = [int(math.floor(float(input.size(i + 2)) * scale_factors[i])) for i in range(dim)]
         scale_factors = None
 
-    if antialias and not (mode in ("bilinear", ) and input.ndim == 4):
-        raise ValueError("Anti-alias option is only supported for bilinear mode")
+    if antialias and not (mode in ("bilinear", "bicubic") and input.ndim == 4):
```

Comment:
Just curious, is there a reason to have the `input.ndim == 4` check here? Both bilinear and bicubic modes will throw an error elsewhere if the input isn't 4-D, right?

Reply:
It comes from this review: https://github.com/pytorch/pytorch/pull/65142#discussion_r710329297

Reply:
Oh, I see why this makes sense, nevermind